pkg/services/cloudprovider/csi.go

```diff
@@ -557,10 +555,9 @@ func LivenessProbeForCSIControllerContainer(image string) corev1.Container {
 
 func VSphereSyncerContainer(image string) corev1.Container {
 	return corev1.Container{
-		Name:            "vsphere-syncer",
-		Image:           image,
-		Args:            []string{"--v=4"},
-		ImagePullPolicy: corev1.PullAlways,
```

Comment:
@moonek - You can override the image to point to a private registry, so `PullAlways` shouldn't matter. Unless you are pre-baking in the images into the OVAs ?

Reply:
@yastij  yes, but in the case of a tls based private registry, I need to put the imagepullsecret somewhere.

Reply:
@yastij there may be other ways, but first I wanted to remove this restriction.

Reply:
/hold cancel